.circleci/config.yml

```diff
@@ -1,5 +1,12 @@
 version: 2
 
+aliases:
```

Comment:
Can you change the circleci version to `2.1` and use [commands](https://circleci.com/docs/2.0/configuration-reference/#commands-requires-version-21) instead?

Also, I believe that rust automatically installed an env script that you can source instead. So you can do `source ~/.cargo/env instead I think.